resumable_train.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Could you please place this in the `scripts/ai2-internal/` directory as well? Our external users won't generally need it.

Reply:
Done.